scripts/job.star

```diff
@@ -0,0 +1,34 @@
+load (
+    'scripts/lib.star',
+    'scan_docker_image_unkown_low_medium_vulnerabilities_step',
+    'scan_docker_image_high_critical_vulnerabilities',
+)
+def cronjobs(edition):
+    if edition != 'oss':
+        edition='grafana-enterprise'
+    else:
+        edition='grafana'
+
+    trigger = {
+        'event': 'cron',
+        'cron': 'nightly',
+    }
+    platform_conf = {
+        'os': 'linux',
+        'arch': 'amd64',
+    }
+    steps=[ 
+        scan_docker_image_unkown_low_medium_vulnerabilities_step(edition),
+        scan_docker_image_high_critical_vulnerabilities(edition),
+    ]
+    return [
+        {
+            'kind': 'pipeline',
+            'type': 'docker',
+            'platform': platform_conf,
+            'name': 'scan-docker-image',
+            'trigger': trigger,
+            'services': [],
+            'steps': steps,
+        }
+    ] 
```

Comment:
Can you ensure that the file has a final newline (GitHub diff points out that it's missing)? It's a convention that source code files should end with a newline.